polars/polars-lazy/src/frame/parquet.rs

```diff
@@ -34,6 +36,57 @@ impl Default for ScanArgsParquet {
     }
 }
 
+/// When async is enabled run the scan in multiple threads.
+///
+/// Note: this is only effective for a small number of files, in the low tens, since it takes about one second to fetch the info from a given file.
+/// Other systems manipulating multiple parquet files rely on a centralized metadata mechanism:
+/// <https://github.com/dask/community/issues/234#issuecomment-1094887172>
+///
+#[cfg(feature = "async")]
+fn scan_parquet_async(url: &str, args: ScanArgsParquet) -> PolarsResult<Vec<LazyFrame>> {
+    use futures::{stream, StreamExt, TryStreamExt};
+    use tokio::runtime::Runtime;
+
+    let rt = Runtime::new()?;
+
+    rt.block_on(async {
+        let cloud_options = args.cloud_options.clone();
+        // List all the files in the bucket.
+        let files = async_glob(url, cloud_options.as_ref()).await?;
+        if files.len() > 10 {
+            println!(
```

Comment:
We should not print the stdout.

Reply:
Makes sense. The intent is to give some feedback for what is likely a slow operation.

I changed this to stderr, but if this is not acceptable I am happy to remove.